external-docs/docs/guides/pivoted-shape.md

```diff
@@ -0,0 +1,105 @@
+# Pivoted Shape
+
+> This page describes how to build and configure a pivoted shape data cube. For more information about the data shapes supported by csvcubed, see [Shaping your data](./shape-data.md). The instructions below assume a basic understanding of writing a [qube-config.json file](./configuration/qube-config.md).
+
+## Single Measure
+
+The [standard shape](./standard-shape.md) is flexible but has a lot of redundancy which can often be removed by using the more concise pivoted form. Our data set on the distribution of the number of 'Arthur's Bakes' stores can be expressed in the pivoted shape as follows:
+
+| Year | Location  | Number of 'Arthur's Bakes' | Status      |
+|:-----|:----------|---------------------------:|:------------|
+| 2022 | London    |                         35 | Provisional |
+| 2021 | Cardiff   |                         26 | Final       |
+| 2020 | Edinburgh |                         90 | Final       |
+| 2021 | Belfast   |                          0 | Final       |
+
+Note that this shape doesn't require that you add any additional columns to the underlying [common structure](./shape-data.md#common-structure); however it does require a different (and explicit) [qube-config.json](./configuration/qube-config.md) configuration; we must ensure that the measure and corresponding unit are attached to the _observations_ column:
+
+```json
+{
+    "$schema": "http://purl.org/csv-cubed/qube-config/v1",
+    "title": "'Arthur's Bakes' stores in UK cities from 2020 to 2022",
```

Comment:
Given this title introduced some problems when inspecting the output, could you remove the `'` quotes?

And in the other examples too.